androidProjects.yml

```diff
@@ -0,0 +1,10 @@
+name: androidProjects
```

Comment:
Could this be named something without reverting to camelcasing?

Something like `Beginner-Level-Android-Studio-Apps`?

Reply:
I'm unable to change the file, the edit option is blocked.

Reply:
@anshulkhattar the fork has been deleted